.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Release JRE Version
+
+# Release on tagged versions
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  build:
+    uses: yetanalytics/runtimer/.github/workflows/runtimer.yml@cf8c77d465f08962cd7901005b8cf197a6149ac1
+    with:
+      java-version: '11'
+      java-distribution: 'temurin'
+      java-modules: 'java.base,java.logging,java.naming,java.xml,java.sql,java.transaction.xa,java.security.sasl,java.management'
```

Comment:
So it seems like we want to keep two different versions of the workflow - one that can be used in other projects, and the other that creates releases in this repo?

Reply:
Well I didn't see any reason to break existing stuff, and didn't want to dup code, so I made this call its own reusable workflow :)

Reply:
Okay, though you might want to add a comment saying that this is mainly for backwards compatibility. (Also there's a linebreak missing after this line which you may want to add back in.)

Reply:
I don't really see it as backwards compatibility or something I even want to remove. For any time you're NOT on gh actions and want a canned jre for instance

Reply:
See documentation in https://github.com/yetanalytics/runtimer/pull/4/commits/61d2fc8d8b70c527ad4275f6900fd7dfd827ea27